.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding this file will mess with developers with a different environment, please consider removing it.

Reply:
sry was a bit lazy and forget it. Normally android studio fixes itself ;-) We should add this file to the ignore list ;-)